### ISSUE TITLE
The list in the repository manager does not inform screen reader users of its status

### ISSUE THREAD

**Issue by FelipeZanabria** [cfillion/reapack#53]
As the title says, there is no way to know what state a repository is in. If all are enabled it is easy to deduce what happens when you press the space key. However, if one does not remember which repositories were disabled, it is difficult to restore them.
The solution I propose is to add a third column, which informs what state a repository is in, and optionally, a combo box similar to the one in the package explorer, to filter the list: Show all, enabled or disabled.
Perhaps you can add a select all button, one to enable whatever is selected in the list, and one to disable.
Thanks for the reply.

**Comment by cfillion** [comment]
Does REAPER's FX chain window (which uses the same icon for toggling bypass) have the same problem?

**Comment by FelipeZanabria** [comment]
Yes and no. When you focus on an fx, the list name changes from List1 to active. If you disable an fx, the list is renamed to disabled. These changes are not reported by the screen reader, because what is focused on is an element within that list (the fx), because what is renamed is the list, and not the list element (the fx).
When outside of the fx list, for example on an effect slider or some other button in the fx chain, turning off an fx, turning off or activating the delta option, the focused control changes its name, because Osara uses this method to present your messages to the user. In this case the state of the fx is handled by osara. I don't have much programming knowledge, but I think that Osara consults some information that the Reaper API provides, to know the status of something. Because, for example, Reaper does not have a list, which is a standard Windows control, to place the different tracks in the main window. Instead, it's a control with an unknown role, which has text that says Track view. Osara uses some Reaper API command or variable to determine which hint or item to focus on.
After giving all that explanation, I would say that this case is not a problem, since there are ways to know the state of an fx. Apart from this, if an effect makes no or very noticeable changes to the audio, when activating the delta option you may hear a silence, indicating that an effect is not deactivated or turned off.
